CRM/Utils/Geocode/Google.php

```diff
@@ -82,7 +82,7 @@ public static function format(&$values, $stateName = FALSE) {
       $add .= ',+';
     }
 
-    if (!empty($values['state_province']) || !empty($values['state_province_id'])) {
+    if (!empty($values['state_province']) || (!empty($values['state_province_id']) && $values['state_province_id'] != 'null')) {
```

Comment:
If I ran into this, I would want to know why `state_province_id='null'` before merging this PR.